Tasks/Npm/task.json

```diff
@@ -55,6 +60,13 @@
             "type": "filePath"
         },
         {
+            "groupName": "advanced",
```

Comment:
We should probably default this to true. But I'll defer to Jovana/Brian who are the npm experts. We default it to true for NuGet.